cassandra/src/cassandra/core.clj

```diff
@@ -136,6 +136,13 @@
                    (set/difference @(:decommissioned test))
                    shuffle))))
 
+(defn seed-nodes
+  "Get a list of seed nodes"
+  [test]
+  (if (= (:rf test) 1)
```

Comment:
Is it OK if replication factor is 1 in some case ? For you local test ?

Reply:
Basically, it is for 1 node cluster.
Default replication factor is 3 and a user can set the parameter freely.
And, I don't want to constrain these parameters. In some cases, a user wants to set these parameters to cause inconsistency intentionally as I did before.